static/app/components/assigneeSelector.spec.jsx

```diff
@@ -317,6 +324,40 @@ describe('AssigneeSelector', () => {
     expect(screen.getByTestId('assignee-selector')).toHaveTextContent('JB');
   });
 
+  it('shows the correct toast for assigning to a non-team member', async () => {
+    jest.spyOn(GroupStore, 'get').mockImplementation(() => GROUP_2);
+    const addMessageSpy = jest.spyOn(IndicatorStore, 'addMessage');
+
+    render(<AssigneeSelectorComponent id={GROUP_2.id} />);
+    act(() => MemberListStore.loadInitialData([USER_1, USER_2, USER_3, USER_4]));
+
+    assignMock = Client.addMockResponse({
+      method: 'PUT',
+      url: `/issues/${GROUP_2.id}/`,
+      statusCode: 400,
+      body: {detail: 'Cannot assign to non-team member'},
+    });
+
+    expect(screen.getByTestId('suggested-avatar-stack')).toBeInTheDocument();
+
+    await openMenu();
+    expect(screen.queryByTestId('loading-indicator')).not.toBeInTheDocument();
+    expect(screen.getByText(`#${TEAM_1.slug}`)).toBeInTheDocument();
+    expect(await screen.findByText('Suggested')).toBeInTheDocument();
+
+    const options = screen.getAllByTestId('assignee-option');
+    expect(options[4]).toHaveTextContent('JD');
+    act(() => userEvent.click(options[4]));
+
+    await waitFor(() => {
+      expect(addMessageSpy).toHaveBeenCalledWith(
+        'Cannot assign to non-team member',
```

Comment:
Here is where we test that we get the expected error `Cannot assign to non-team member` in the frontend